sagemaker-debugger/tensorflow_profiling/callback_bottleneck.ipynb

```diff
@@ -76,7 +76,7 @@
    "cell_type": "markdown",
```

Comment:
<p>nit: change header to "3." for consistency with 1 and 2.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>fixed</p>